### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by feng-tao**
update the stale slack link

### CODE REVIEW

**Comment by feng-tao** [comment]
cc @danwom @jinhyukchang @ttannis

**Comment by jornh** [comment]
LGTM - one thought though: If the Slack token has plans to go stale more often it would be more DRY (don't-repeat-yourself) to just link from that section to the badges section at https://github.com/lyft/amundsenfrontendlibrary#amundsen .. maybe?

**Comment by feng-tao** [comment]
Good idea @jornh :) PR updated :)

File: README.md [hunk 1]
```diff
@@ -44,7 +44,7 @@ Please note that the mock images only served as demonstration purpose.
 ## Get Involved in the Community
 
 Want help or want to help? 
-Join our [slack channel](https://bit.ly/2tFsN1X) and our [mailing list](https://groups.google.com/forum/#!forum/amundsen-dev)
+Join our [slack channel](https://github.com/lyft/amundsenfrontendlibrary#amundsen) and our [mailing list](https://groups.google.com/forum/#!forum/amundsen-dev)
```
Comment (ttannis) [inline README.md]: I think it's misleading that a link titled `slack channel` links back to our github page. I support the url only being in one spot though, so I think here we should give clear instruction like:
`Use the button in our [header](https://github.com/lyft/amundsenfrontendlibrary#amundsen) to join our slack channel...`

Comment (feng-tao) [reply README.md]: updated

**Pull request merged**